Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.6-alpine
+
+RUN apk --no-cache add g++
```

Comment:
Please, group the build dependencies and delete their at the end of Dockerfile

Reply:
Grouping build dependencies yes. Deleting them I'm not sold on. 

Yes it shrinks the image by quite a bit, but it also reduces usability by others. This isn't ever supposed to be "production" grade simply due to it being a testing lib. The extra ~140MB of g++ and associated packages seems a reasonable price for facilitating the use of Locust for an audience that isn't as familiar with Docker or Alpine. Hell by that measure I should probably toss in bash and eat the extra 40MB.

Reply:
I appreciate @spayeur207's comment, but cutting the size in half would be great for those who would like to widely distribute workers. Maybe you could make it delete the build-only items, and add a note on how devs could comment-out that behavior. Then it would be nice for non-devs and (arguably) still good enough for advanced devs.